docs/deployment_practices.md

```diff
@@ -74,7 +74,7 @@ In your SSL (port 443) virtual host, set up HSTS and use these settings to give
     SSLProtocol all -SSLv2 -SSLv3
     SSLHonorCipherOrder on
     SSLCompression off
-    SSLCipherSuite EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:EECDH+RC4:RSA+RC4:!MD5
```

Comment:
Asking around on [Twitter](https://twitter.com/garrettr_/status/563792813393842178) confirmed that discontinuing RC4 would prevent some people with very old clients from accessing the page, but that it is a fairly minor concern.